src/auto-trace.helper.js

```diff
@@ -25,6 +25,7 @@ export function wrapObjectWithError(err, stacktraceErr, extraContext) {
 				errOut.message = err;
 			}
 			else {
+				errOut.originalErrorObject = err;
```

Comment:
Attach the original object to the error so that it can be accessed through `originalErrorObject`